pkg/registry/apis/folders/register_test.go

```diff
@@ -27,9 +27,6 @@ func TestFolderAPIBuilder_Validate_Create(t *testing.T) {
 		name        string
 	}
 
-	initialMaxDepth := folderValidationRules.maxDepth
-	folderValidationRules.maxDepth = 2
-	defer func() { folderValidationRules.maxDepth = initialMaxDepth }()
```

Comment:
^^ this pattern is not great.  the maxDepth is now passed as an argument to the thing that will fetch parents